lua/nvim-treesitter/_meta/parsers.lua

```diff
@@ -0,0 +1,21 @@
+---@meta
+error('Cannot require a meta file')
+
+---@class InstallInfo
+---@field path? string
```

Comment:
This is a good place to add documentation.

Reply:
Such as? Happy to do so, but would appreciate concrete suggestions :)

Reply:
Oh, you mean the fields themselves, as in
```lua
-@field path? string the path
```

Reply:
Yes, then we get the docs in LSP hover.

Reply:
Can also be:

```lua

--- some desc
--- @field path? string
```